roles/util-cfg-service/templates/ui/de.properties.j2

```diff
@@ -204,3 +204,11 @@ org.iplantc.discoveryenvironment.notification.amqp.exchange.auto-delete = {{ amq
 ###############################################################################
 org.iplantc.discoveryenvironment.analysis.support.user = siuser
 org.iplantc.discoveryenvironment.support = {{ terrain.base }}/support-email
+
+###############################################################################
+# DE Session Store Settings
+###############################################################################
+org.iplantc.discoveryenvironment.redis.host = {{ redis_host }}
+org.iplantc.discoveryenvironment.redis.port = {{ redis_port }}
+org.iplantc.discoveryenvironment.redis.db.number = 0
+org.iplantc.discoveryenvironement.redis.password = {{ redis_password }}
```

Comment:
This definitely needs to be added to `de.properties.tmpl` on the UI pr, now that I can see it's being added here. The UI's on consul configs everywhere as of tomorrow. Should (hopefully) be fairly self-explanatory how to add them to that file.